src/main/app/fact/FactApi.ts

```diff
@@ -1,9 +1,11 @@
 import { Logger } from '../../types/Logger';
 import {AxiosError, AxiosInstance} from 'axios';
+import {OpeningTime, OpeningType} from '../../types/OpeningTime';
 
 export class FactApi {
 
   private readonly baseURL = '/courts';
+  private readonly adminBaseUrl = '/admin/courts';
```

Comment:
Seems that this file caters for a lot of different types of calls; admin opening, general, courts...types 

Is there a way to split this out into different files? I only mention this as it will grow and grow with time. If its not possible, then thats fair :) Just thought I'd mention

Reply:
This is a fair point and something I thought too. I wonder if we should refactor this class in a separate ticket. The existing methods all do pretty much the same thing (only the url changes) so could be improved. Error handling could be improved too. We'd need to test all endpoints if we refactor, hence I'm thinking we deal with this separately.